tools/db/README.md

```diff
@@ -0,0 +1,121 @@
+### Configure datastore
+
+Before you can build and deploy OpenWhisk, you must configure a backing datastore. The system supports any self-managed [CouchDB](using-couchdb) instance or
+[Cloudant](using-cloudant) as a cloud-based database service.
+
+#### Using CouchDB
+
+If you are using your own installation of CouchDB, make a note of the host, port, username and password. Then within your `openwhisk` directory, copy the file `template-couchdb-local.env` to `couchdb-local.env` and edit as appropriate. Note that:
+
+   * the username must have administrative rights
+   * the CouchDB instance must be accessible over `https` (although the certificate does not need to be valid)
```

Comment:
addressed with content from #413 
* the CouchDB instance must be accessible over `http` or `https` (the latter requires a valid certificate)